### DISCUSSION TITLE
Why export-ignore files are included in vendor folder while using 'dist'?

### DISCUSSION THREAD

**Discussion by ProgrammerZ** [composer/composer#11817] [Q&A / Support] [answered]
I have the following config in my `composer.js`

```
"config": {
     "optimize-autoloader": true,
     "preferred-install": "dist",
      "discard-changes": true,
      "sort-packages": true,
      "allow-plugins": {
          "pestphp/pest-plugin": true,
          "php-http/discovery": true
      }
  },
```

But I still have all tests included in packages in Vendor dir. For example although Laravel framework package includes `.gitattributes` file with `/tests export-ignore` I still see all the tests in `/test`

I have even tried to do composer update `--prefer-dist` with same results

Composer  V2.6.6.

**Comment by Seldaek** [comment, marked as answer]
We'd need to see from where the package was installed. Maybe if you paste the full output of a `composer install` (after removing the vendor/laravel folder at least so it reinstalls those) it'll be clearer.

**Reply by ProgrammerZ** [reply]
When I have re-installed it today, everything installed as expected. I do not know what is the reason, since my steps today were absolutely the same and have been repeated multiple times in the past. (remove `vendor` folder and re-install with `composer install --prefer-dist`). Thank you.

**Reply by totoprayogo1916** [reply]
I am also having the same problem. This only applies when using a custom repository, .gitattributes not working.
